addons/binding/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/cli/Parser.java

```diff
@@ -0,0 +1,399 @@
+/**
```

Comment:
Could this class be moved to an internal package or is there a need to export it?